crev-data/src/proof/mod.rs

```diff
@@ -259,13 +259,11 @@ impl Proof {
                                 self.body += line;
                                 self.body += "\n";
                             }
+                        } else if is_signature_line(line) {
```

Comment:
I actually disagree with this lint in clippy. Oftentimes it makes it less-clear what's going on, when there are multiple nested logical choices. Eample:

if A {

} else { 
   if B {
   } else {
   }
}

B is a sub-case of "not A" and by flattening them, some information is lost, IMO. But not a blocker.

Reply:
Ideally, we could disable that lint globally in the project.

Reply:
I see what you mean, I'll disable it globally.

Reply:
There isn't currently a way of disabling this particular lint check globally, see: https://github.com/EmbarkStudios/rust-ecosystem/issues/22

I will instead skip this lint for this PR.

Reply:
We could just add it in every affected crate.